test/integration/api_test.exs

```diff
@@ -129,11 +130,12 @@ defmodule ExRabbitPool.Integration.ApiTest do
           spawn(fn ->
             ExRabbitPool.with_channel(pool_id, fn {:ok, channel} ->
               :ok = AMQP.Channel.close(channel)
+              :timer.sleep(500)
```

Comment:
we want to trigger then `{:EXIT, channel_pid, :normal}` message before we check the channel back to the pool